### PR TITLE
[KAJ-1019] eLux Distant VDI: documentation updates

### PR DESCRIPTION
**Pull request by pegnovi**
Elux documentation updates for release 1.6.1.

### CODE REVIEW

File: eLux/README.md [hunk 1]
```diff
@@ -18,74 +16,87 @@ DigiCert SHA2 Assured ID Code Signing CA:
...
-This section allows to indicate which certificate files to load.
-
-The name of a section and an indication wether it is On or Off needs to be use here. The section must contain the File and the Trustargs attributes.
-
-#### Sample
-[RibbonRTC]
-
+### 3. Configuration
+
+#### 3.1 Setup
```
Comment (jfmoron) [inline eLux/README.md:25]: you can remove this whole section, all the setup is already done on Elux.

File: eLux/README.md [hunk 2]
```diff
@@ -18,74 +16,87 @@ DigiCert SHA2 Assured ID Code Signing CA:
...
+`VirtualDriver=Thinwire3.0, TWI, SmartCard, SSPI, TUI, KandyDistant.PlugIn`
+4. Under the same section (`[ICA 3.0]`), add a key for our plugin and assign it the value of "On":
+`KandyDistant.PlugIn=On`
+5. Add a section for our plugin (and optionally set your custom executable path which defaults to `/usr/lib/rtc`):
+```
+[KandyDistant]
+ExecutablePath = /your/path/
+```
+
+#### 3.2 Modifying Citrix Configuration
```
Comment (jfmoron) [inline eLux/README.md:43]: Note this is also in the Kandy ini as I explained today.

File: eLux/README.md [hunk 3]
```diff
@@ -18,74 +16,87 @@ DigiCert SHA2 Assured ID Code Signing CA:
...
+LogPath and LogLevel can be added and modified under the KandyDistant section.
+
+ex:
+```
+[KandyDistant]
+LogPath  = ~/Kandy/logs
```
Comment (jfmoron) [inline eLux/README.md:49]: log path cannot be changed at the moment

File: eLux/README.md [hunk 4]
```diff
@@ -18,74 +16,87 @@ DigiCert SHA2 Assured ID Code Signing CA:
...
+- trace
+
+
+#### 3.3 KandyLib Configuration
+KandyLib-specific configuration can be set and modified in your `kandy.ini` file which is expected to be found in `/setup` directory.
+
+The `KandyDistant` section allows configuration flags that affect the browser container to be set.
+Note that the legacy section, `RibbonRTC`, will still work but `KandyDistant` will take priority if both sections contain the same flag.
+
+- CachePath: location where to store the application cache, defaults to: ~/Library/Application Support/Kandy/cache.
```
Comment (jfmoron) [inline eLux/README.md:72]: that cache path default location is for Mac, on Elux you should verify the default in the code but it should be somewhere in /tmp

File: eLux/README.md [hunk 5]
```diff
@@ -18,74 +16,87 @@ DigiCert SHA2 Assured ID Code Signing CA:
...
-[CACERT1]
-
-File=/setup/cacerts/CACERT1.pem
-
-Trustargs=C,,
-
-[INTERMEDIATECERT1]
-
-File=/setup/cacerts/INTERMEDIATECERT1.pem
+### 4. Before Running
```
Comment (jfmoron) [inline eLux/README.md:96]: this is already done for Elux with our script

**Review by jfmoron — CHANGES_REQUESTED** [review]

File: eLux/README.md (hunk 3 above)
Comment (pegnovi) [reply eLux/README.md:49]: Will remove mention of log path.

File: eLux/README.md (hunk 1 above)
Comment (pegnovi) [reply eLux/README.md:25]: Will remove.

File: eLux/README.md (hunk 2 above)
Comment (pegnovi) [reply eLux/README.md:43]: I will move this.

File: eLux/README.md (hunk 4 above)
Comment (pegnovi) [reply eLux/README.md:72]: Forgot to update this. I found the default yesterday.

**Review by jfmoron — CHANGES_REQUESTED** [review]

File: eLux/README.md [hunk 6]
```diff
@@ -18,74 +16,52 @@ DigiCert SHA2 Assured ID Code Signing CA:
...
-Trustargs=C,,
-
-[INTERMEDIATECERT1]
-
-File=/setup/cacerts/INTERMEDIATECERT1.pem
+### 4. Logs
+The logs can be found at `/var/log/kandy/`.
 
-Trustargs=,,,
+Congratulations! You have built, signed, installed and configured the KandyDistant plugin for Citrix Workspace App on Elux!
```
[the remark this answers is not in the document]
Comment (pegnovi) [reply eLux/README.md]: removed

File: eLux/CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,11 @@ Kandy Distant Driver for VDI eLux change log.
 - This change log follows [keepachangelog.com](http://keepachangelog.com/) recommendations.
 - This project adheres to [Semantic Versioning](http://semver.org/), however, eLux packages also include a -X identifier, which represents the version of the eLux package
 
+## 1.6.1 - 2022-02-9
+### Changed
```
[the remark this answers is not in the document]
Comment (pegnovi) [reply eLux/CHANGELOG.md:9]: As discussed, Known issues and unreleased have been put in the README instead of the changelog.

**Review by jfmoron — APPROVED** [review]

**Pull request merged**